src/database/models/comments.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable require-jsdoc */
+/* eslint-disable no-unused-vars */
+
+const { Model } = require('sequelize');
+
+module.exports = (sequelize, DataTypes) => {
+  class Comments extends Model {
+    static associate(models) {
+      models.Users.hasMany(Comments, {
+        foreignKey: {
+          name: "UserId",
```

Comment:
This is "userId" camel case as from your migrations. Same case for an accommodation